test/extended/prometheus/prometheus_ocm_workqueue.go

```diff
@@ -0,0 +1,85 @@
+package prometheus
+
+import (
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/ibmcloud"
+)
+
+var _ = g.Describe("[Feature:Prometheus][Feature:Builds] Prometheus", func() {
```

Comment:
Not sure if it is appropriate to add this to the `Conformance` suite. We need to make sure that we never break metrics reported by OCM. `Feature:Builds` will only run in the `e2e-*-build` suites.

Reply:
meant to switch to Conformance and forgot ... as I think you are, given the broad range of workqueues (networking, deployment configs, images stuff, builds, templates) not sure what else

and sig-instrumentation is maybe the only other choice, but that seems less right ... of course chime in if you think differently

will update to Conformance here shortly in the interim just in case

Reply:
actually sig-instrumentation is better than [Feature:Prometheus]

will do that and conformance

Reply:
no, sig-* are master/4.5 labels :-)

just switch builds to conformance